gc/base/standard/ConfigurationGenerational.hpp

```diff
@@ -64,8 +74,18 @@ class MM_ConfigurationGenerational : public MM_ConfigurationStandard
 	bool initialize(MM_EnvironmentBase* env);
 	MM_MemorySubSpaceSemiSpace *createSemiSpace(MM_EnvironmentBase *envBase, MM_Heap *heap, MM_Scavenger *scavenger, MM_InitializationParameters *parameters, UDATA numaNode = UDATA_MAX);
 	virtual void tearDown(MM_EnvironmentBase* env);
+	/**
+	 * Sets the number of gc threads
+	 *
+	 * @param env[in] - the current environment
+	 */
+	virtual void initializeGCThreadCount(MM_EnvironmentBase* env);
```

Comment:
Use the correct formatting for the function description: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly
```suggestion
	/**
	 * Sets the number of GC threads
	 *
	 * @param[in] env the current environment
	 * @return void
	 */
	virtual void initializeGCThreadCount(MM_EnvironmentBase* env);
```